scripts/packages/chromium-package.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
+# Installing chromium 32 bit or 64 bit browser 
+if [ "$arch" == "i386" ];
+then 
+wget -q security.ubuntu.com/ubuntu/pool/universe/c/chromium-browser/chromium-browser_71.0.3578.98-0ubuntu0.16.04.1_i386.deb -O chromium-browser_i386.deb
```

Comment:
Won't `sudo apt-get -qq -y install chromium-browser:i386` work?
I am seeing an hard coded version number on the url and I guess this won't install the latest 32 bit package **if** a new update comes out

Reply:
@abishekvashok I will test one of the releases with the following command and get back
I was also thinking of another alternative that could work

Reply:
Okay @pranav1698